pkg/deploy/elbv2/load_balancer_synthesizer.go

```diff
@@ -55,7 +62,13 @@ func (s *loadBalancerSynthesizer) Synthesize(ctx context.Context) error {
 	// I don't like this, but it's the easiest solution to meet our requirement :D.
 	for _, sdkLB := range unmatchedSDKLBs {
 		if err := s.lbManager.Delete(ctx, sdkLB); err != nil {
-			return err
+			errMessage := err.Error()
+			if strings.Contains(errMessage,"OperationNotPermitted") {
```

Comment:
Is it possible to check for more specific error?

Reply:
I can check if the error message contains keyword like "deletion protection", but not sure if it's better than check the error code as it is now. Or I can check both.

Reply:
As far as I can see, the error message is `"OperationNotPermitted: Load balancer 'arn:aws:elasticloadbalancing:xyz123' cannot be deleted because deletion protection is enabled\n\tstatus code: 400, request id: abc123"`
So I changed to check both keywords "OperationNotPermitted" and "deletion protection".